### DISCUSSION TITLE
Windsurf

### DISCUSSION THREAD

**Discussion by clmarshall** [modelcontextprotocol/.github#126] [Q&A] [answered]
### Pre-submission Checklist

- [X] I have checked that this question would not be more appropriate as an issue in a specific repository
- [X] I have searched existing discussions and documentation for answers

### Question Category

- [ ] Protocol Specification
- [ ] SDK Usage
- [ ] Server Implementation
- [X] General Implementation
- [ ] Documentation
- [ ] Other

### Your Question

Is there a procedure to install MCPs with Codeium Windsurf?

I've attempted the "Add a tool" method but it insists on adding it to my current workspace

**Comment by andrewlayer** [comment]
From what I can tell, Windsurf has not implemented MCP into their client application.  Here are the [list of clients](https://modelcontextprotocol.io/clients) that have.

**Comment by leonid-deriv** [comment]
are there plans to integrate MCP? Ok, probably this question is in the wrong place :)

**Comment by Mehdi-Bl** [comment]
You should ask Windsurf team over this is Anthropic and MCP spec team.

**Comment by tadasant** [comment, marked as answer]
Windsurf has indeed integrated MCP and the docs are here: https://docs.codeium.com/windsurf/mcp#mcp-integration

As other commenters have mentioned, recommend you take this discussion (e.g. troubleshooting) to their help forums.

**Reply by leonid-deriv** [reply]
Is it available in Team version as well?
